src/lang/builtins_thread.ml

```diff
@@ -22,8 +22,10 @@
 
 open Lang_builtins
 
+let () = Lang.add_module "thread"
+
 let () =
-  add_builtin "thread.run.recurrent" ~cat:Control
+  add_builtin "thread.run_recurrent" ~cat:Control
```

Comment:
I see.. Is it possible to remove or define a module then (see comment in the corresponding `.liq` file..)